packages/core-utils/src/state.js

```diff
@@ -0,0 +1,151 @@
+import isEqual from "lodash.isequal";
```

Comment:
Is this `state.js` file used anywhere?

Reply:
Yes, query.js has this:
`import { getActiveSearch } from "./state";`

Reply:
Oh, I was just confused because I didn't see any imports in this PR.